dist.ini

```diff
@@ -0,0 +1,44 @@
+name = AnyEvent-RabbitMQ
+author = Masahito Ikuta <cooldaemon@gmail.com>
+license = Perl_5
+copyright_holder = AnyEvent-RabbitMQ's developers
+
+[@Filter]
+-bundle = @Basic
+-remove = Readme
+
+[Metadata]
+x_contributors = Tom Doran <bobtfish@cpan.org>
+x_contributors = Nicolas R <atoomic@cpan.org>
+x_contributors = Dave Lambley <dlambley@cpan.org>
+x_contributors = Ruslan Zakirov <ruz@bestpractical.com>
+x_contributors = Masahito Ikuta <cooldaemon@gmail.com>
+x_contributors = Rod Taylor <rod.taylor@gmail.com>
+x_contributors = Carl Hörberg <carl@cloudamqp.com>
+
+[InstallGuide]
+;[CPANFile]
+[VersionFromModule]
```

Comment:
Dist::Zilla does this by default, no need to a plugin.

Reply:
That doesn't seem to work on my (Ubuntu 18.04) machine,
```
$ dzil build
[DZ] beginning to build AnyEvent-RabbitMQ
[DZ] no version was ever set
[DZ] no version was ever set at inline delegation in Dist::Zilla for logger->log_fatal (attribute declared in /usr/share/perl5/Dist/Zilla.pm at line 766) line 18.
```

Reply:
Sorry, I mixed it up! I'm using PkgVersion so I only need to set the version in dist.ini and it gets added to each module.

Reply:
I've not used PkgVersion, but patches welcome of course!